### ISSUE TITLE
Single pin

### ISSUE THREAD

**Issue by Soth-Eng** [jgeisler0303/QTouchADCArduino#1]
i made this work on a single arduino pin what do you recomend on this since you define 2 pins in your code.
if you whant i can make a arduino library for it so its easier to use.
you may contact me directly.

**Comment by jgeisler0303** [comment]
The method used requires always at least two pins in order to charge the adc sample and hold capacitor and the actual probe capacitor to different potentials. Once you have more than one probe, one of the currently not measured probes' pin can be used for this purpose. Id you you only want one probe, make sure the other pin is not used for anything else, it has to be there an be free!!

Thanks for your offer to make a library. I think someone else did this already here: http://playground.arduino.cc/Code/ADCTouch﻿ but I didn't test it.

**Comment by colinives** [comment]
Hi there and thanks for this code - can you say if it will work on the attiny85?

Thank you

**Comment by jgeisler0303** [comment]
Yes, it should work on attiny. As far as I know the ADC units are pretty much the same.
Let us know if you succeed.
